main/integration-tests/README.md

```diff
@@ -14,25 +15,37 @@ To run integration tests, the following resources need to be created in advance:
 
 #### Resources to create through SWB UI:
 
-* **Test Administrator:** Create an internal admin-role user for running integration tests. (**Note the username and password**)
+- **Test Administrator:** Create an internal admin-role user for running integration tests. (**Note the username and password**)
 
-* **Test Project:** Create a default project for running integration tests. (**Note the projectId**)
+- **Test Project:** Create a default project for running integration tests. (**Note the projectId**)
 
-* **AWS Budget:** Create a budget for the AWS account associated with the Test Project.
+- **AWS Budget:** Create a budget for the AWS account associated with the Test Project.
+
+#### Resources for Advanced Tests
+
+- **Environment Type:** Import each specific Service Catalog Product as an Environment Type
+
+- **Environment Configuration:** Create a configuration for every corresponding environment type that was created
+
+- **Data Source Study:** If testing BYOB studies, create an external BYOB study on the deployment with Read & Write Access
 
 #### Resources to create through AWS Console:
 
-* **Test Administrator password:** Create an AWS Parameter Store record to store Test Administrator password. Pick a parameter name of your choice. Use 'SecureString' for parameter type 'text' for data type.(**Note the parameter name**)
+- **Test Administrator password:** Create an AWS Parameter Store record to store Test Administrator password. Pick a parameter name of your choice. Use 'SecureString' for parameter type 'text' for data type.(**Note the parameter name**)
 
-* **Cost Explorer:** Turn on Cost Explorer service, activate tags 'Env', 'Proj' anc 'createdBy'.
+- **Cost Explorer:** Turn on Cost Explorer service, activate tags 'Env', 'Proj' anc 'createdBy'.
 
 ### Config File
 
 Once test resources are created, create a config file `main/integration-tests/config/settings/<STAGE>.yml` follow the example `main/integration-tests/config/settings/example.yml`.
 Use the same `<STAGE>` name as the main config file in `/main/config/settings`.
 
 Use Test Admin username, projectId, and Parameter Store name noted from the [Test Resources](#test-resources) section for field username, passwordPath and projectId in the config file.
-Use the same config values used in `/main/config/settings` for fields awsRegion, awsProfile, solutionName, envName, ane envType. 
+Use the same config values used in `/main/config/settings` for fields awsRegion, awsProfile, solutionName, envName, ane envType.
+
+For each `{workspaceType}EnvTypeId` and `{workspacetype}ConfigId`, use the ids for the specific Environment Type and Environment Configuration created as Test Resources.
```

Comment:
For each `{workspaceType}EnvTypeId` and `{workspacetype}ConfigId`, use the IDs for specific environment type and environment configuration created as test resources.